### DISCUSSION TITLE
need help for the ADR220-B175 data

### DISCUSSION THREAD

**Discussion by smallduan** [sandialabs/WecOptTool#254] [General]
Hi 

I was trying to follow the tutorial3 and want to know how to set the generator data accoding to the  ADR-B SERIES manual（given by the tutorials）. Can someone give me the data of ADR220-B175.

Best regards

**Comment by rgcoe** [comment]
@smallduan - Thanks for your interest. I think you'll need to reach out to [Akribis (https://akribis-sys.com/)](https://akribis-sys.com/) for a copy of the spec. sheet, as our copy is marked confidential.
